packages/ref-metrics-ilamb/src/cmip_ref_metrics_ilamb/standard.py

```diff
@@ -143,43 +208,49 @@ def run(self, definition: MetricExecutionDefinition) -> MetricResult:
             csv_file = definition.to_output_path(f"{source_name}.csv")
             ref_file = definition.to_output_path("Reference.nc")
             com_file = definition.to_output_path(f"{source_name}.nc")
+            log_file = definition.to_output_path(f"{source_name}.log")
+            log_id = logger.add(log_file, backtrace=True, diagnose=True)
```

Comment:
Nice soln to this. We could make this a context manager one day to make sure it is cleaned up and could also be used by the other packages